Assets/Scripts/Interfaces/IHealthBar.cs

```diff
@@ -0,0 +1,8 @@
+using UnityEngine;
+
+public interface IHealthBar
+{
+    IHealthSystem HealthSystem { set; }
+    void SetSize(float health);
+    void SetColour(Color color);
```

Comment:
Perhaps a longer description could be better to fully understand what the method does, because from this naming it is not clear

Reply:
OK.I will rename it in the future